packages/mdc-ripple/foundation.js

```diff
@@ -266,6 +267,9 @@ class MDCRippleFoundation extends MDCFoundation {
     });
   }
 
+  /**
```

Comment:
I would suggest:

```js
activate(event = null) {
  this.activate_(event);
}
```

And the same for deactivate.

I'd prefer avoiding single-letter variable names because they tend to be harder to quickly understand as well as search.